src/main/java/com/faforever/client/legacy/GameStatus.java

```diff
@@ -0,0 +1,20 @@
+package com.faforever.client.legacy;
+
+import com.faforever.client.legacy.domain.GameState;
+
+public enum GameStatus {
+  NONE, HOST, LOBBY, PLAYING;
+
+  //FIXME cannot resolve gamestatus from gamestate
+  public static GameStatus getFromGameState(GameState gameState) {
+    switch (gameState.getString()) {
+      case "playing":
+        return GameStatus.PLAYING;
+      case "open":
+        return GameStatus.LOBBY;
+      //unknown and closed
+      default:
+        return GameStatus.NONE;
+    }
+  }
+}
```

Comment:
I'm not resolving gamestatus string, I'm resolving a gamestate string, so I don't see the point in resolving gamestatus strings. Also, gamestate only returns unknown, playing, open and closed. There is no way to resolve host from the gamestate, that is determined here
https://github.com/micheljung/downlords-faf-client/pull/103/files#diff-0cfcc15ad2ff76d2d4b8f7f674002a48R91